### PR TITLE
Mj add support for amp multivariant testing

### PR DESCRIPTION
**Pull request by michaelbjacobson**
Enables multivariant testing for AMP epic tests

### CODE REVIEW

File: conf/routes [hunk 1]
```diff
@@ -62,7 +62,7 @@ POST  /frontend/apple-news-epic-tests/archive                  controllers.epic.
 GET   /frontend/apple-news-epic-tests/archive/:testName        controllers.epic.AppleNewsEpicTestArchiveController.get(testName: String)
 GET   /frontend/apple-news-epic-tests/archived-test-names      controllers.epic.AppleNewsEpicTestArchiveController.list
 
-# ----- AMO epic 'tests' ----- #
+# ----- AMP epic 'tests' ----- #
```
Comment (TomPretty) [inline conf/routes:65]: Good spot!

File: public/src/components/channelManagement/epicTests/epicTestVariantEditor.tsx [hunk 2]
```diff
@@ -183,7 +183,7 @@ const EpicTestVariantEditor: React.FC<EpicTestVariantEditorProps> = ({
             }
             onBlur={handleSubmit(onSubmit)}
             name="highlightedText"
-            label="Hightlighted text"
+            label="Highlighted text"
```
Comment (TomPretty) [inline public/src/components/channelManagement/epicTests/epicTestVariantEditor.tsx:186]: 🙈

**Review by TomPretty — APPROVED** [review]
Looks good to me 👍

**Review by tomrf1 — APPROVED** [review]
🎉

**Pull request merged**